packages/renderer-protocol/proto/RendererProtocol.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+message CRDTManyMessages {
+  string sceneId = 1;
+  bytes	payload = 2;
+}
+
+message CRDTResponse {}
+
+message CRDTStreamRequest {}
+
+service CRDTService {
+  rpc SendCRDT(CRDTManyMessages) returns (CRDTResponse){}
+  rpc CRDTNotificationStream(CRDTStreamRequest) returns (stream CRDTManyMessages) {}
```

Comment:
You may be interested in this little chunk of comments: https://github.com/decentraland/rpc/blob/main/src/client.ts#L109-L111